proposed/http-factory/http-factory.md

```diff
@@ -34,7 +34,10 @@ in separate classes.
 Has the ability to create client requests.
 
 ```php
-namespace Psr\Http\Message;
+namespace Psr\Http\Factory;
```

Comment:
Question: where was the discussion behind the new namespace? I'd typically expect these to be in the same namespace, as they're related to the _messages_ in particular; having them in a `Factory` subnamespace might cause conflicts in the future (e.g., if we ever did a "middleware factory" PSR — which I doubt will happen, but it serves as an example).